tutor/specs/components/navbar/__snapshots__/index.spec.jsx.snap

```diff
@@ -93,7 +93,10 @@ exports[`Main navbar renders and matches snapshot 1`] = `
         />
         <span
           title="Menu and settings"
-        />
+        >
+          Hi 
+          
+        </span>
          
         <span
```

Comment:
Glad you figured out how to update the snapshots - I just realized I'd never mentioned them to you 😁